examples/simple-example/simple-example-camunda-7/src/main/resources/bpmn/message-c7.bpmn

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
⛏️ @rami-nk - do we want to add a custom "retries" property to the simple-example process?

Reply:
Would make sense to add it to any of other examples. We could add it to the simple-example, but we also should be a bit careful not to add too much to it so it becomes less simple. But for now it should be ok I say

Reply:
We should add it to its documentation (README). At least reference the other readme